portfolio/src/main/java/com/google/sps/servlets/Constants.java

```diff
@@ -0,0 +1,6 @@
+package com.google.sps.servlets;
+
+public final class Constants {
+  private Constants() {}
+  public static int DEFAULT_COMMENTS_DISPLAYED = 10;
+}
```

Comment:
newline missing. you might want to double check your github config. it should add these newlines automatically for you.